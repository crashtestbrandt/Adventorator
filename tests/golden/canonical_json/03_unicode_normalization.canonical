{"café":"naïve","key_é":"value_ë"}