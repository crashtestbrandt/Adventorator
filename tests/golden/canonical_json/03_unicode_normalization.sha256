64b2304b49bf1d763e242d954bb66b07fd24bb63789703f08c75adc4bfcf2cbb
