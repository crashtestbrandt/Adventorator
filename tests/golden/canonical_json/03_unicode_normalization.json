{
  "caf\u00e9": "na\u00efve",
  "key_\u00e9": "value_\u00eb"
}